recipe/build.sh

```diff
@@ -16,6 +16,7 @@ autoreconf -fi
 ./configure --prefix="${PREFIX}" \
   ${CONFIGURE_ARGS} \
   --with-hdf5="${PREFIX}" \
+  --with-metis="${PREFIX}" \
   --enable-shared \
   --enable-tools \
   --enable-pymoab \
```

Comment:
```suggestion
  --enable-pymoab \
  --disable-blaslapack \
```
Will force only Eigen3 for MOAB

Reply:
Agreed. Perhaps we can make this specific modification when we are adding Windows builds for Conda (without TempestRemap dependency). All TR dependent workflows needed for E3SM do require BLASLAPACK. No way around it.

Reply:
If this is the case, don't we need blaslapack as a dependency as well?